pandas/tests/indexes/test_engines.py

```diff
@@ -61,7 +61,13 @@ class TestTimedeltaEngine:
     @pytest.mark.parametrize(
         "scalar",
         [
-            pd.Timestamp(pd.Timedelta(days=42).asm8.view("datetime64[ns]")),
+            # error: Argument 1 to "Timestamp" has incompatible type "timedelta64";
+            # expected "Union[integer[Any], float, str, date, datetime64]"
+            pd.Timestamp(
+                pd.Timedelta(days=42).asm8.view(
+                    "datetime64[ns]"
+                )  # type: ignore[arg-type]
+            ),
```

Comment:
This is a numpy issue. can't see an issue on github, maybe fixed in master. will check soon.